python_modules/dagster/dagster/_core/definitions/observable_asset.py

```diff
@@ -0,0 +1,21 @@
+from dagster._core.definitions.asset_spec import ObservableAssetSpec
+from dagster._core.definitions.decorators import asset
+
+
+def create_unexecutable_observable_assets_def(observable_asset_spec: ObservableAssetSpec):
```

Comment:
update to take `AssetSpec`, maybe error on ignored properties but i dont think silent omission is unreasonable